sdk/src/packet.rs

```diff
@@ -35,6 +35,8 @@ bitflags! {
         const ROUND_COMPUTE_UNIT_PRICE = 0b0010_0000;
         /// For tracking performance
         const PERF_TRACK_PACKET  = 0b0100_0000;
+        /// For marking packets from staked nodes
+        const FROM_STAKED_NODE = 0b1000_0000;
```

Comment:
out of packet flags 😢

Reply:
If necessary, can remove `ROUND_COMPUTE_UNIT_PRICE`, but I still hope that feature would be approved some day.

Reply:
we don't serialize the bitfield anywhere do we? next flag just makes it a `u16`